pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -210,6 +210,7 @@ func TestGetTransactionConfirmations_Integration(t *testing.T) {
 				})
 			}
 
+			time.Sleep(time.Second)
```

Comment:
If we want to commit this blasphemy, let's add a code comment referencing https://github.com/checksum0/go-electrum/issues/10.

Reply:
a0dd61e521b69b1c855244d0d170521d17ded30e